integrations/azure/README.md

```diff
@@ -0,0 +1,944 @@
+    "title": "Storage Container",
+    "icon": "S3",
+    "schema": {
+      "properties": {
+        "publicAccess": {
+          "title": "Public Access",
+          "type": "string"
+        },
+        "hasImmutabilityPolicy": {
+          "title": "Has Immutability Policy",
+          "type": "boolean"
+        },
+        "hasLegalHold": {
+          "title": "Has Legal Hold",
+          "type": "boolean"
+        },
+        "deleted": {
+          "title": "Deleted",
+          "type": "boolean"
+        },
+        "deletedTime": {
+          "title": "Deleted Time",
+          "type": "string"
+        },
+        "remainingRetentionDays": {
+          "title": "Remaining Retention Days",
+          "type": "number"
+        },
+        "leaseStatus": {
+          "title": "Lease Status",
+          "type": "string"
+        },
+        "leaseState": {
+          "title": "Lease State",
+          "type": "string"
+        },
+        "defaultEncryptionScope": {
+          "title": "Default Encryption Scope",
+          "type": "string"
+        },
+        "version": {
+          "title": "Version",
+          "type": "string"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "storageAccount": {
+        "target": "storageAccount",
+        "title": "Storage Account",
+        "required": false,
+        "many": false
+      }
+    }
+  }
+```
+
+</details>
+
+<details>
+<summary>port-app-config.yaml</summary>
+
+```yaml
+  - kind: Microsoft.Storage/storageAccounts/blobServices/containers
+    selector:
+        query: "true"
+        # azure resource api version to query
+        apiVersion: '2023-01-01'
+    port:
+      entity:
+        mappings:
+          # lower case the resource group namespace and name to align with other azure resources
+          identifier: '.id | split("/") | .[3] |= ascii_downcase |.[4] |= ascii_downcase | join("/")'
+          title: .name
+          blueprint: '"storageContainer"'
+          properties:
+            publicAccess: .properties.publicAccess
+            hasImmutabilityPolicy: .properties.hasImmutabilityPolicy
+            hasLegalHold: .properties.hasLegalHold
+            deleted: .properties.deleted
+            deletedTime: .properties.deletedTime
+            remainingRetentionDays: .properties.remainingRetentionDays
+            leaseStatus: .properties.leaseStatus
+            leaseState: .properties.leaseState
+            defaultEncryptionScope: .properties.defaultEncryptionScope
+            version: .properties.version
+          relations:
+            # lower case the resource group namespace and name to align with other azure resources
+            storageAccount: '.id | split("/") | .[3] |= ascii_downcase | .[4] |= ascii_downcase | .[:-4] |join("/")'
+```
+
+</details>
+
+
+## Adding a new Azure Resource kind
+
```

Comment:
I suggest adding a summary of the steps that need to be taken (blueprint, mapping, auth, etc)